components/scream/src/control/tests/sc_tests.cpp

```diff
@@ -33,13 +33,7 @@ TEST_CASE ("surface_coupling")
   ekat::Comm comm (MPI_COMM_WORLD);
 
   // The random numbers generator
-  std::random_device rd;
-  const unsigned int catchRngSeed = Catch::rngSeed();
-  const unsigned int seed = catchRngSeed==0 ? rd() : catchRngSeed;
-  if (comm.am_i_root()) {
-    std::cout << "seed: " << seed << (catchRngSeed==0 ? " (catch rng seed was 0)\n" : "\n");
-  }
-  rngAlg engine(seed);
+  auto engine = setup_random_test(&comm);
   RPDF pdf(0.0,1.0);
```

Comment:
@jgfouca As of yesterday evening there is a new test case "recreate_mct_coupling" in this file on master that uses the same RNG. So it will need this change as well.

Reply:
@tcclevenger , thanks for the heads-up. I've done another upstream merge and changed this test to use setup_random_test.